### DISCUSSION TITLE
InvalidStateError

### DISCUSSION THREAD

**Discussion by Virgil-N** [wailsapp/wails#885] [Q&A]
Build with firebug mode.
Use axios send http request console print an error: InvalidStateError.
Message: response error:  InvalidStateError: The object is in an invalid state. {...}.
What's this?
How to solve it?

OS: mac book pro

**Comment by leaanthony** [comment]
This is a Javascript error. StackOverflow would be a better place to ask.

**Reply by Virgil-N** [reply]
But I use javascript sending request works.

**Reply by Virgil-N** [reply]
Got error when I use build app.

**Reply by leaanthony** [reply]
Then please open an issue with reproducable steps
